src/register/RegistrationFields/CountryField/CountryField.jsx

```diff
@@ -49,7 +55,7 @@ const CountryField = (props) => {
         { target: { name: 'country' } },
         { countryCode, displayValue: countryDisplayValue },
       );
-    } else if (!selectedCountry.countryCode) {
+    } else if (!selectedCountry.displayValue) {
```

Comment:
why did we change this?

Reply:
if we put some wrong value inside the country field and we move around the login and signup tabs then this !selectedCountry.countryCode becomes always true and onChangeHandler set the country field with empty data as we are calling this onChangeHandler with blank data so this reset the country field, actual behaviour is to keep whatever user have input in country field should be there in tabs switch